backend/src/main/java/com/redhat/cloud/notifications/templates/Rhosak.java

```diff
@@ -0,0 +1,79 @@
+package com.redhat.cloud.notifications.templates;
+
+import com.redhat.cloud.notifications.models.EmailSubscriptionType;
+import io.quarkus.qute.CheckedTemplate;
+import io.quarkus.qute.TemplateInstance;
+
+// Name needs to be "Rhosak" to read templates from resources/templates/Rhosak
+public class Rhosak implements EmailTemplate {
+    private static final String SCHEDULED_UPGRADE = "scheduled_upgrade";
+    private static final String DISRUPTION = "disruption";
+    private static final String INSTANCE_CREATED = "instance_created";
+    private static final String INSTANCE_DELETED = "instance_deleted";
+    private static final String ACTION_REQUIRED = "action_required";
+
+    @Override
+    public TemplateInstance getTitle(String eventType, EmailSubscriptionType type) {
+        if (eventType.equals(SCHEDULED_UPGRADE)) {
+            return Templates.scheduledUpgradeTitle();
+        } else if (eventType.equals(DISRUPTION)) {
+            return Templates.serviceDisruptionTitle();
+        } else if (eventType.equals(INSTANCE_CREATED)) {
+            return Templates.instanceCreatedTitle();
+        } else if (eventType.equals(INSTANCE_DELETED)) {
+            return Templates.instanceDeletedTitle();
+        } else if (eventType.equals(ACTION_REQUIRED)) {
+            return Templates.actionRequiredTitle();
+        }
+        return null;
```

Comment:
@josejulio Returning `null` won't cause any issue in the calling code, but I can see other templates throwing an exception here. What's your preference?

Reply:
Good point, technically we should not arrive here. I think I'll go with the exception for consistency.

Reply:
Fixed.